### PR TITLE
updated aws library

### PR DESCRIPTION
**Pull request by sf-kansara**
Updated s3 library.

References:
https://github.com/aws/aws-sdk-js-v3
https://github.com/aws/aws-sdk-js-v3#configuration
https://aws.amazon.com/blogs/developer/modular-aws-sdk-for-javascript-is-now-generally-available/

### CODE REVIEW

File: src/providers/aws-s3.provider.ts [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
+import {S3} from '@aws-sdk/client-s3';
 import {AwsS3Config, AWSS3Bindings} from '../types';
 
-export class AwsS3Provider implements Provider<AWS.S3> {
+export class AwsS3Provider implements Provider<S3> {
   constructor(
     @inject(AWSS3Bindings.Config)
     private readonly config: AwsS3Config,
   ) {
-    AWS.config.update(Object.assign({}, {signatureVersion: 'v4'}, this.config));
```
Comment (samarpan-b) [inline src/providers/aws-s3.provider.ts:10]: This is needed. we need to define signatureversion

**Review by samarpan-b — CHANGES_REQUESTED** [review]

**Comment by SouravSF** [comment]
@sf-kansara Can you confirm if this works on local????

**Comment by akshatdubeysf** [comment]
@ashutosh-bansal-2136

**Comment by w20k** [comment]
Don't you need to add `region` to `AWSS3Bindings.Config`? Or I'm missing something and change docs, also :)

**Comment by ashutosh-bansal-2136** [comment]
Hi @w20k,
This PR is stale actually, if u have any other issues then u can tell us at [this](https://github.com/sourcefuse/loopback4-s3/pull/12).

**Comment by akshatdubeysf** [comment]
Closing this as it has been fixed in another PR

**Pull request closed without merging**